packages/@aws-cdk/aws-appsync/test/integ.api-import.ts

```diff
@@ -0,0 +1,67 @@
+/// !cdk-integ *
+
+import * as path from 'path';
+import * as db from '@aws-cdk/aws-dynamodb';
+import * as cdk from '@aws-cdk/core';
+import * as appsync from '../lib';
+
+/*
+ * Creates an Appsync GraphQL API in a separate stack.
+ * Add dependencies to imported api.
+ *
+ * Stack verification steps:
+ * Install dependencies and deploy integration test. Check if data sources are
+ * connected to the graphQL Api
+ *
+ * -- cdk deploy --app 'node integ.api-import.js' stack            -- start         --
+ * -- aws appsync list-graphql-apis                                -- obtain api id --
+ * -- aws appsync list-data-sources --api-id [api_id]              -- testDS/None   --
+ * -- cdk destroy --app 'node integ.api-import.js' stack baseStack -- clean         --
```

Comment:
nit: i don't think the deploy/destroy need to be covered - just the steps to verify the expected configuration.

Reply:
there are two stacks so maybe its important? im fine with removing it though

Reply:
it's a nit, so up to you!